test/models/onnx/space_to_depth.prototxt

```diff
@@ -10,6 +10,11 @@ graph {
       i: 2
       type: INT
     }
+    attribute {
+      name: "mode"
+      s: "blocks_first"
+      type: STRING
+    }
```

Comment:
In the ONNX specification `SpaceToDepth` operator does not have an attribute called `mode`.
This model is invalid